portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,48 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Victor Alvarez - Home</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body>
+    <!-- Navigation bar from W3schools.com -->
+     <ul>
+        <li class="active"><a href="index.html">About Me</a></li>
+        <li><a href="resume.html">Resume</a></li>
+    </ul>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Victor Alvarez's Portfolio</h1>
+
+      <img src="/images/promPhoto.jpeg" alt="image of myself" style="display: block; margin-left: auto; margin-right: auto; height: 50%; width: 50%; border-radius: 50%;"/>
+      
+      <p>Victor Alvarez is a student of Harvard College, class of 2023. He is pursuing a double major in computer science and mathematics.
+        Although there is no clear plan, he knows he wants a career where he will apply his skills and continue to learn about both of these
+        fields. <br/> 
+        Victor has several interests outside of academics. He likes to play soccer, listen to music, play video games and watch television.
+        Aside from the few activities named, Victor is willing to try new things as he's always ready to find a new interest!
+      </p>
+      
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
```

Comment:
Maybe I missed it, but does href="/hello" go anywhere?

Reply:
It doesn't but this was included in the source code so i assumed we would need this later. I just left it as is for now.